src/main/java/io/nats/client/Nats.java

```diff
@@ -470,4 +487,63 @@ public static Options defaultOptions() {
         }
         return list;
     }
+
+    /**
+     * Sets the message delivery thread pool size.
+     *
+     * By default, each asynchronous subscription has its own thread
+     * responsible to dispatch the subscription's messages.
+     * If the application needs to create a high number of subscriptions,
+     * this may not scale.
+     *
+     * Setting a message delivery thread pool size will limit the
```

Comment:
Maybe add a note the effect that size can be reset to grow, but cannot be reduced?